packages/vitest/src/node/reporters/utils.ts

```diff
@@ -0,0 +1,37 @@
+import type { Reporter } from '../../types'
+import { ReportersMap } from './index'
+import type { BuiltinReporters } from './index'
+
+async function loadCustomReporterModule<C extends Reporter>(path: string): Promise<new () => C> {
+  let customReporterModule: { default: new() => C }
+  try {
+    customReporterModule = await import(path)
```

Comment:
Maybe we should what globalSetup does? Otherwise users cannot use typescript and other features. 

It's better if we load the module with ViteNode, I think.

Reply:
I'll take a look and see what I can do 👍🏻

Reply:
@sheremet-va I've revised the code by moving the ViteNodeRunner out of GlobalSetup and into VItest to support loading both TS and JS files in both ESM and CommonJS formats. The ViteNodeRunner is exposed through the `runner` property/field on Vitest so that GlobalSetup still has access to it.

Also, should we consider updating the docs to mention support for custom reporters via the CLI? If so, should I reach out to anyone before making those changes?

Reply:
Docs are located in docs folder. To run it, use `pnpm run docs:dev`, I think?
Just add enough information there and we are good 👍

Reply:
@sheremet-va sorry for the wait. Just pushed up a small change to the docs for command line options. Let me know if more needs to be added or if it's not quite clear. Thanks